doc/bible.md

```diff
@@ -101,14 +101,29 @@ Those running for election must declare in which region they intend to operate t
 
 New candidates are eligible to submit a candidacy for a single notary node in any of the four regions in their first election. Participation in the pre-season testnet is <b>highly recommended</b> to learn the basics and demonstrate capability.
 
-Notary nodes achieving a regional top 3 ranking at the ranking snapshot date are automatically re-elected to operate that node in the next season. 
+Notary nodes achieving a regional top 3 ranking at the ranking snapshot date are **automatically re-elected to operate that node (same region)** in the next season, and can apply for an **one spot in any region** as long as you have no more than two nodes in a region, or six in total.
 
-Notary nodes achieving a regional top 5 ranking are eligible to submit a candidacy for an additional notary node for each region they held a top 5 position in at the ranking snapshot date.
+Notary nodes achieving a regional ranking of 4 or 5 are eligible to apply for an **two spots without any restriction on which region(s)** as long as you have no more than two nodes in a region, or six in total.
 
-Notary nodes who are not in their regional top 5 at the end of the season can reapply for a single node for each of the region(s) they served in during the previous season.
+Notary nodes who are not in their regional top 5 at the end of the season can apply for a **one spot without any restriction on which region(s)** as long as you have no more than two nodes in a region, or six in total.
+
+Newcomers can apply **only for 1 spot in any region**.
 
 <b>No notary can hold more than 2 nodes in a single region, or more than 6 nodes in total.</b>
 
```

Comment:
May be good idea is also add _cheatsheet_ here:

1. TOP 3 keeps their spot and can apply for **1 spot** in **any** region.
2. RANK 4-5 can *apply* for **2 spots** per owned node in **any** region.
3. Below TOP 5 can *re-apply*  for  **1 spot** per owned node in **any** region.
4. Newcomers can apply **only for 1 spot**  in **any** region.
5. No notary can hold more than 2 nodes in a single region, or more than 6 nodes in total.

Bcz above there are still too much words, from my point of view - short form of cheatsheet will not be superfluous.

Reply:
I like this version with easy to understand.

Reply:
thanks, updated